src/EditorFeatures/Core/Implementation/InlineRename/Taggers/ClassificationTypeDefinitions.cs

```diff
@@ -24,13 +22,13 @@ internal sealed class ClassificationTypeDefinitions
         [ClassificationType(ClassificationTypeNames = InlineRenameField)]
         [Name(InlineRenameField)]
```

Comment:
Is this the display text?  Is it localized now that you're making it user visible?